### DISCUSSION TITLE
API is down

### DISCUSSION THREAD

**Discussion by MSSIDDIQUE** [lukePeavey/quotable#88] [Archived]
It looks like the API might be down right now.

**Comment by sheshnathiicmr** [comment]
+1

**Comment by dpios1** [comment]
Happening to me as well.

**Comment by xmartgithub** [comment]
Same happening to me as well.

**Comment by looper-ops** [comment]
Not working for me as well.

**Comment by jimmyxp01** [comment]
I'm facing the same problem. Location -India

**Comment by lukePeavey** [comment]
Ill look into it

**Comment by lukePeavey** [comment]
## Resolved

This has been fixed. Sorry for the inconvenience everyone.

**Comment by xmartgithub** [comment]
Thank you @Luke

**Comment by lukePeavey** [comment]
@MSSIDDIQUE Thanks for reporting this.

**Comment by sheshnathiicmr** [comment]
@lukePeavey Thanks for this wonderful resource and free service that somehow we Indian loves :D, Maybe you can use https://www.statuscake.com to monitor downtime, in the free version we can monitor one URL, you see here also I am using free one :D

**Comment by jimmyxp01** [comment]
@lukePeavey It's still not working.
My code:
```
import requests
from random_user_agent.user_agent import UserAgent
from random_user_agent.params import SoftwareName, OperatingSystem

software_names = [SoftwareName.CHROME.value]
operating_systems = [OperatingSystem.WINDOWS.value, OperatingSystem.LINUX.value]   
user_agent_rotator = UserAgent(software_names=software_names, operating_systems=operating_systems, limit=100)

r= requests.get("http://api.quotable.io/random",headers={"User-Agent":user_agent_rotator.get_random_user_agent()},allow_redirects=True)
```
I'm getting this error Error 
```
requests.exceptions.ConnectionError: HTTPConnectionPool(host='api.quotable.io', port=80): Max retries exceeded with url: /random (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002ACDF33D2E0>: Failed to establish a new connection: [Errno 11002] getaddrinfo failed'))
```

**Comment by lukePeavey** [comment]
@jimmyxp01 do you get a response when you call the API directly in your browser?

**Comment by lukePeavey** [comment]
The downtime was caused by an expired DNSSEC key, which expired on July 11. 

It was fixed on July 12 (last night). The API should be working now. 


Both of these sites are showing that the API is up now. 

[isitdownrightnow](https://www.isitdownrightnow.com/api.quotable.io.html)

[downforeveryoneorjustme](https://downforeveryoneorjustme.com/api.quotable.io)

**Comment by jimmyxp01** [comment]
@lukePeavey a Few hours ago it's open in the browser and gives a JSON response back. But In python, it's failed.
At this time API is down in my location (INDIA). But When I use VPN it's working properly. At this time without a VPN it's not working in browser or python.

**Comment by jimmyxp01** [comment]
@lukePeavey  
response: `requests.exceptions.ConnectionError: HTTPConnectionPool(host='api.quotable.io', port=80): Max retries exceeded with url: /random (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000297D932BA60>: Failed to establish a new connection: [Errno 11002] getaddrinfo failed'))`

**Reply by lukePeavey** [reply]
@jimmyxp01 Is the API working for you now?  If not, let me know.

**Reply by i-am-ijaz** [reply]
It is working fine.

**Comment by lukePeavey** [comment]
@jimmyxp01 It can take up to 72 hours for DNS changes to propagate worldwide.  Based on your location, it might take a little longer for the changes to take effect.

Hopefully the API should be working again for you shortly.  If not, let me know. 

Again, sorry for the inconvenience.
